hw/ip/pattgen/data/pattgen_testplan.hjson

```diff
@@ -79,5 +79,19 @@
       milestone: V2
       tests: ["pattgen_stress_all"]
     }
+    {
+      name: stress_all_with_reset
+      desc: '''
+            Combine above sequences in one test then randomly select for running.
+
+            Stimulus:
+              - Randomly add reset within the sequences then switch to another one
+
+            Checking:
+              - All sequences should be finished and checked by the scoreboard
+      '''
+      milestone: V2
+      tests: ["pattgen_stress_all_with_rand_reset"]
+    }
```

Comment:
you can import testplan for this 
"hw/dv/tools/dvsim/testplans/stress_all_with_reset_testplan.hjson"

Reply:
thanks, I remove the test description then add .hjson file